replication/logic/diff/diff.go

```diff
@@ -88,8 +88,20 @@ func SortVersionListByCreateTXGThenBookmarkLTSnapshot(fsvslice []*FilesystemVers
 	return sorted
 }
 
+func FilterBookmarksFromVersionList(fsvslice []*FilesystemVersion) []*FilesystemVersion {
+	fslice := make([]*FilesystemVersion, 0)
+	for _, fv := range fsvslice {
+		if fv.Type != FilesystemVersion_Bookmark {
+			fslice = append(fslice, fv)
+		}
+	}
+	return fslice
+}
+
 func IncrementalPath(receiver, sender []*FilesystemVersion) (incPath []*FilesystemVersion, conflict error) {
 
+	receiver = FilterBookmarksFromVersionList(receiver)
```

Comment:
1. Naming: let's call this `StripBookmarksFromVersionList` 
2. Please add a comment to explain why we need to strip the bookmarks.